src/components/ex-downloadlist/ex-downloadlist.scss

```diff
@@ -0,0 +1,43 @@
+.downloadlist {
+  @apply w-full p-5 my-8;
+
+  background: #f5f8fa;
+
+  @screen md {
+    @apply p-8 my-6;
+  }
+
+  .downloadlist__header {
+    @apply font-body text-gray-0 relative pl-12 font-normal leading-5;
+
+    &::before {
+      @apply absolute left-0 bg-center bg-no-repeat bg-contain;
+
+      content: '';
+      height: 3rem;
+      width: 2rem;
+      background-image: url('./resources/images/sicherearbeit-icon-download.svg');
+
+      @screen md {
+        @apply mr-8;
+      }
+    }
+
+    .downloadlist__title {
+      @apply text-l font-semibold uppercase;
+    }
+
+    .downloadlist__subtitle {
+      @apply text-s mt-2;
+    }
+  }
+
+  .downloadlist__elements {
+    @apply flex flex-col pl-0 mt-4 mb-0 list-none;
```

Comment:
There is a padding-bottom on the ul that can be removed as it makes the y-spaces look uneven:

![image](https://user-images.githubusercontent.com/88790097/134131385-19249d72-6c91-411e-8748-1262640768d8.png)

Reply:
Removed the space